cluster/config.go

```diff
@@ -66,3 +66,11 @@ func getPublishNotReadyAddress() bool {
 	}
 	return pubNotReady
 }
+
+func getMinIOImage() string {
+	concreteMinIOImage := "minio/minio:RELEASE.2019-09-26T19-42-35Z"
```

Comment:
why such an old version ? `RELEASE.2019-12-19T22-52-26Z  ` this is the latest

Reply:
it was a msitake D: